nimcfg.json

```diff
@@ -0,0 +1,23 @@
+{
+    "comments": {
+        "lineComment": "#",
+        "blockComment": [ "#[", "]#" ]
+    },
+    "brackets": [
+        ["[", "]"],
+        ["(", ")"]
+    ],
+    "autoClosingPairs": [
```

Comment:
I dont sure that this is good idea to use json configuration together with code based in `nimMain.ts`, also `autoClosingPairs` is marked as `Deprecated` in https://github.com/Microsoft/vscode/blob/master/src/vs/vscode.d.ts#L2786
May be it will more reasonable use declaration in code for better type safety?